src/esp/sim/BatchReplayRenderer.cpp

```diff
@@ -274,6 +278,24 @@ void BatchReplayRenderer::doRender(
                  "a standalone renderer", );
 
   renderer_->draw(framebuffer);
+
+  if (debugLineRender_) {
+    framebuffer.bind();
+    constexpr unsigned envIndex = 0;
+    auto projCamMatrix = renderer_->camera(envIndex);
+    debugLineRender_->flushLines(projCamMatrix, renderer_->tileSize());
+  }
+}
+
+esp::geo::Ray BatchReplayRenderer::doUnproject(
+    unsigned envIndex,
+    const Mn::Vector2i& viewportPosition) {
+  // temp stub implementation: produce a placeholder ray that varies with
+  // viewportPosition
+  return esp::geo::Ray(
```

Comment:
Another shortcut here. We can implement this properly later. As is, this will be useful for helping @0mdc integrate the batch replay renderer into the SIRo Python app.